src/database/models/Youfocus/Playlist.ts

```diff
@@ -0,0 +1,26 @@
+import { Schema, model, models, Model } from 'mongoose';
+import {PlaylistModel} from '@/interfaces';
+
+// Define the schema
+const PlaylistSchema = new Schema<PlaylistModel>(
+  {
+    playlistUrl: {
+      type: String,
+      required: [true, 'Playlist URL is required'],
+    },
+    videos: [
+      {
+        title: String,
+        videoId: String,
+        thumbnail: String,
+      },
+    ],
```

Comment:
Create a Separate Interface for Videos. Check how other Models are created which contain array like structure